cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -370,17 +464,49 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Expand Into",
       text =
-        """When both the start and end node have already been found, expand-into is used to find all connecting relationships between the two nodes.""".stripMargin,
+        """When both the start and end node have already been found, the `Expand(Into)` operator is used to find all relationships connecting the two nodes.""".stripMargin,
```

Comment:
Since both sides of the relationship are already in scope, the side with the smallest degree will be used. This can make a big difference when dense nodes can show up as end points.

Reply:
👍